docs/getting-started-guides/scratch.md

```diff
@@ -212,7 +212,7 @@ A Kubernetes binary release includes all the Kubernetes binaries as well as the
 You can use a Kubernetes binary release (recommended) or build your Kubernetes binaries following the instructions in the
 [Developer Documentation](../devel/README.md).  Only using a binary release is covered in this guide.
 
-Download the [latest binary release](https://github.com/GoogleCloudPlatform/kubernetes/releases/latest) and unzip it.
+Download the [latest binary release](https://releases.k8s.io/latest) and unzip it.
```

Comment:
redirects to https://github.com/kubernetes/kubernetes/tree/latest (and thus broken)